### PR TITLE
Patch 3

### PR DESCRIPTION
**Pull request by Aaron-Junker**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

Added the logo and a banner. I have the rights. See microsoft/PowerToys#7178

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/powertoys/index.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ short_description: Microsoft PowerToys is a set of utilities for power users.
 related: microsoft
 topic: powertoys
 wikipedia_url: https://en.wikipedia.org/wiki/Microsoft_PowerToys
+logo: index.png
 ---
+![banner](https://raw.githubusercontent.com/microsoft/PowerToys/master/doc/images/overview/PT%20hero%20image.png)
```
Comment (BlakeWilliams) [inline topics/powertoys/index.md:12]: Is this something other topics do? If not, can we remove it?

Comment (Aaron-Junker) [reply topics/powertoys/index.md]: I changed it

**Review by damaneice — APPROVED** [review]
Thank you for your contribution

File: topics/powertoys/index.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ short_description: Microsoft PowerToys is a set of utilities for power users.
 related: microsoft
 topic: powertoys
 wikipedia_url: https://en.wikipedia.org/wiki/Microsoft_PowerToys
+logo: index.png
```
Comment (damaneice) [inline topics/powertoys/index.md:10]:
```suggestion
logo: powertoys.png
```

**Review by damaneice — CHANGES_REQUESTED** [review]
Thank you for your contributions. Can you fix the build failure? 

I think all you have to do is rename the logo from `index.png` to `powertoys.png`

**Comment by Aaron-Junker** [comment]
@damaneice Now it can be merged

**Review by iancanderson — APPROVED** [review]
Thanks!

**Pull request merged**